### DISCUSSION TITLE
Best place to ask for assistance with installation?

### DISCUSSION THREAD

**Discussion by itinneed2022** [graphite-project/graphite-web#2759] [Q&A]
I've been trying to install and configure graphite-web, carbon, and whisper on Centos 7 for a couple of days now. I've ran into so many issues with the two main methods (source and pip) that I'm not sure where to even start asking for help I feel like python2's deprecation somehow ended Graphite and I'm just late to the party.

What's the best way to ask questions about the install and configuration? The launchpad page seems like it hasn't been used in awhile.

**Comment by deniszh** [comment]
Hi @itinneed2022 
I'm afraid launchpad page is abandoned. You can ask help here or open issue, but if nobody can help - then how it is, sorry.
